.github/workflows/v1.yml

```diff
@@ -0,0 +1,25 @@
+on:
```

Comment:
I think it would be better to consolidate the two workflows into one, having them executed as separate jobs within the same run, which will reduce the number of runs that appear on our actions page. The workflow would look something like this:

```
on:
  push:
    branches:
      - master
      - actions/**

jobs:
  v1:
    runs-on: ubuntu-latest
    steps:
        ...
  v2:
    runs-on: ubuntu-latest
    steps:
      ...
```

This will run the workflows in parallel jobs and easy to reference within a single run. What do you think?

Reply:
Sure, sounds good to me. I will be closing this PR and will create a new one in the new repo that we discussed in scrum meeting today.